src/components/src/BulkOrder/bulkorder.main.less

```diff
@@ -117,6 +117,11 @@
         .scan-barcode-title {
           margin-right: 5px;
         }
+
+        .form-content-submit {
+          display: inline-block;
+          margin-right: 10px;
```

Comment:
This is good, but the style is missing when the user is logged out.
![image](https://user-images.githubusercontent.com/39800563/73096738-833b6f80-3eb3-11ea-82b6-90dc16fd35b9.png)